app/models/api/sample_io.rb

```diff
@@ -68,6 +68,36 @@ def json_root
     map_attribute_to_json_attribute(:supplier_name)
     map_attribute_to_json_attribute(:donor_id)
     map_attribute_to_json_attribute(:phenotype)
+    map_attribute_to_json_attribute(:sibling)
+    map_attribute_to_json_attribute(:is_resubmitted)
+    map_attribute_to_json_attribute(:date_of_sample_collection)
+    map_attribute_to_json_attribute(:date_of_sample_extraction)
+    map_attribute_to_json_attribute(:sample_extraction_method)
+    map_attribute_to_json_attribute(:sample_purified)
+    map_attribute_to_json_attribute(:purification_method)
+    map_attribute_to_json_attribute(:concentration)
+    map_attribute_to_json_attribute(:concentration_determined_by)
+    map_attribute_to_json_attribute(:sample_type)
+    map_attribute_to_json_attribute(:sample_storage_conditions)
+    map_attribute_to_json_attribute(:genotype)
+    map_attribute_to_json_attribute(:age)
+    map_attribute_to_json_attribute(:cell_type)
+    map_attribute_to_json_attribute(:disease_state)
+    map_attribute_to_json_attribute(:compound)
+    map_attribute_to_json_attribute(:dose)
+    map_attribute_to_json_attribute(:immunoprecipitate)
+    map_attribute_to_json_attribute(:growth_condition)
+    map_attribute_to_json_attribute(:rnai)
+    map_attribute_to_json_attribute(:organism_part)
+    map_attribute_to_json_attribute(:time_point)
+    map_attribute_to_json_attribute(:disease)
+    map_attribute_to_json_attribute(:subject)
+    map_attribute_to_json_attribute(:treatment)
+    map_attribute_to_json_attribute(:genome_size)
+    map_attribute_to_json_attribute(:saphyr)
+    map_attribute_to_json_attribute(:pacbio)
+    map_attribute_to_json_attribute(:date_of_consent_withdrawn)
+    map_attribute_to_json_attribute(:user_id_of_consent_withdrawn)
```

Comment:
For the ones where we want to change the name, it looks like the place to do that is here? https://github.com/sanger/unified_warehouse/blob/develop/app/models/sample.rb

Reply:
I'd vote for performing the translation in Sequencescape, rather than the warehouse, isolates the weirdness to one code-base.

Reply:
So looking at existing translations:
```
id: :id_sample_lims,
uuid: :uuid_sample_lims,
```
Feel like they belong in the WH, as they deal with specific WH implementation.

In an ideal world I guess:
```
      sample_common_name: :common_name,
      sample_description: :description,
      sample_ebi_accession_number: :accession_number,
      sample_taxon_id: :taxon_id,
      sample_public_name: :public_name,
      sample_sra_hold: :sample_visibility,
      sample_strain_att: :strain
 ```

Would all belong in Sequencescape world, as it feels like they are dealing with SS implementation decisions. (It mostly comes from time when the WH was actually the v0.5 API. But that is no longer exposed elsewhere, so we're relatively safe making changes)

One thing I will note with translations, is that you can still supply a message to the WH with the translated term, and it will work. So its actually not too tricky to fix these issues.